### ISSUE TITLE
Add CKAN ini file to version control

### ISSUE THREAD

**Issue by genkimaps** [genkimaps/gis-hub#67]
Currently, the CKAN ini file is excluded from version control (git) because it contains sensitive info such as passwords. We can alleviate this by storing passwords on the server in environment variables that are loaded at runtime.  Then we can add the ini file to version control. 

See this page for reference:  https://docs.ckan.org/en/2.8/maintaining/configuration.html

**Comment by genkimaps** [comment]
This did not work.  Reason is that it is not really possible to pass environment vars to supervisor, which is the low-level system daemon that initializes CKAN.  Tried these and a few other things, did not work:

https://stackoverflow.com/questions/12900402/supervisor-and-environment-variables
https://stackoverflow.com/questions/34038511/supervisord-using-a-variable-inside-the-supervisord-conf

Forget this for now, it's not super high priority.  I restored the older version of the ini file (with passwords, etc) and restarted CKAN. Closing this issue.

**Comment by genkimaps** [comment]
The idea behind putting the CKAN config file (dfo.ini) in version control is that it's useful to have all those settings in the git repo. However, this file contains passwords, API keys and other sensitive info, which we don't want on git.  In theory, removing the sensitive info from the config file and putting it in environment vars should work. 

I managed to get environment variables working with supervisor. The trick is that any environment variable containing special characters has to be double-quoted:

`environment=
  CKAN_SQLALCHEMY_URL="postgresql://user:pass@localhost/ckan",
  OTHER_ENV_VAR="blah blah"
command=uwsgi --paste config:/home/tk/ckan/dfo.ini ... `

However, if we do this and remove environment vars from the dfo.ini file, then we have three problems:
1. cannot run `paster` commands from the command line 
2. Have to set each enviro var manually in Pycharm if running for debug purposes
3. downstream code in CKAN, such as extensions, that depends on having variables set in the CKAN ini file will not work.  

All of this can be alleviated by adding the enviro vars to the user's profile, to Pycharm, and so on. But this means we are storing the same info in multiple places, which is a recipe for disaster.  Better to keep all the enviro vars defined in only one place: the CKAN ini file, which is used by all parts of CKAN and commands that access CKAN indirectly, such as paster. 

TL;DR: keep the CKAN ini file as-is, using environment vars is too much trouble.
